README.md

```diff
@@ -53,7 +62,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n/2) The stack to parse this must be able to hold half the total number elements as it needs to hold the first half then check it with the second half.
```

Comment:
Two very minor errors, but no deduction:

1. There's no such thing as O(n/2). You always remove all the low order terms (so O(4n<sup>2</sup>+3n-6) is the same as O(4n<sup>2</sup>), and then you remove constant factors (O(n<sup>2</sup>)

2. In your explanation, you assume the parse knows how long the input is, so that it knows when it reaches the half-way point. It can't.
1 \times 2
$$